docs/resources/workspace_deployed_module.md

```diff
@@ -0,0 +1,28 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "tharsis_workspace_deployed_module Resource - terraform-provider-tharsis"
```

Comment:
After thinking about this more I think the best name for this resource is `tharsis_apply_module` which is consistent with our tharsis apply cli/api naming.